package.json

```diff
@@ -158,7 +158,7 @@
   "scripts": {
     "lint": "npm-run-all -p -c lint-js lint-css",
     "lint-js": "eslint .",
```

Comment:
Why this script validates `.tsx` files?
![image](https://user-images.githubusercontent.com/1420883/83689974-96e0aa80-a5f8-11ea-8809-00717da152ab.png)